### DISCUSSION TITLE
openIE 2020.03.26 Specify path to spacy model

### DISCUSSION THREAD

**Discussion by rajiv-mids** [allenai/allennlp#5301] [Q&A]
Hi,

I work in a bank where servers cannot directly connect to the internet and download dependent models due to security restrictions. How do I configure the location of the spacy model. When I run

Predictor.from_path("/path/to/openie-model.2020.03.26.tar.gz') 

It says

Spacy models 'en_core_web_sm' not found downloading. And then it times out. Can I configure path to the spacy model?

Also can I use en_core_web_lg instead?

Thanks!

**Comment by epwalsh** [comment]
I tried `spacy download en_core_web_sm`, and apparently that just calls out to `pip install en-core-web-sm==3.0.0`. So maybe if you just install `en-core-web-sm==3.0.0` in your environment spacy will avoid trying to download anything.
